sdk/react/panels/AudioAndCCSelectionPanel.js

```diff
@@ -150,8 +143,8 @@ class AudioAndCCSelectionPanel extends Component {
     return (
       <ItemSelectionScrollView
         style={styles.panelItemSelectionView}
-        items={itemsWithLocalizedUndefinedLanguage}
-        selectedItem={selectedLocalizedItem}
+        items={this.props.audioTracksTitles}
```

Comment:
Was it an intentional change? I mean it looks out of PIP scope, right?

Reply:
No, it's not my changes and I don't use this snippet. Probable it was inherited from `dev`-branch .